src/Nuget.PackageIndex/ISettingsJson.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+using System.Collections.Generic;
```

Comment:
i kelp this format accros all files since it was automatically added some time ago to all existing files, so i just wanted to be consistent in new files too.